### PR TITLE
Fix link #

### PR DESCRIPTION
**Pull request by fgregoribattista**
If the link does not have the position name(#), add the contract name.

### CODE REVIEW

File: builders/super-builder.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = {
     if (baseContract !== undefined) {
       if(!baseContract.ignore) {
         // If file is not ignored, then add link to file as well
-        builder.push(`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md#${node.name.toLowerCase()})`);
+        builder.push((!node.name)?`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md)`:`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md#${node.name.toLowerCase()})`);
```
Comment (pmprete) [inline builders/super-builder.js]: If we don't have node.name shouldn't we do this?
```suggestion
        builder.push((!node.name)?`[${baseContract.contractName}](${baseContract.contractName}.md)`:`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md#${node.name.toLowerCase()})`);
```

**Review by pmprete — APPROVED** [review]
Looks Good! Nice catch!

File: builders/super-builder.js (hunk 1 above)
Comment (fgregoribattista) [reply builders/super-builder.js]: Yes, we should. Good catch!! 👍🏼

**Review by pmprete — APPROVED** [review]
Great work!

**Pull request merged**